test/LibraryManager.Test/Providers/Cdnjs/CdnjsProviderTest.cs

```diff
@@ -106,70 +106,72 @@ public void Cleanup()
         [TestMethod]
         public async Task InstallAsync_EmptyFilesArray()
         {
-            var desiredState = new LibraryInstallationState
-            {
-                ProviderId = "cdnjs",
-                LibraryId = "jquery@1.2.3",
-                DestinationPath = "lib"
-            };
+            string providerId = "cdnjs";
```

Comment:
Validation no longer happens at provider level, but at Manifest only. All our product calls go through Manifest, so I created the manifest to install from so we can get the proper validation errors. Same for many other tests  below.